README.rst

```diff
@@ -188,12 +188,15 @@ The NeMo Framework can be installed in a variety of ways, depending on your need
   * This is recommended for Automatic Speech Recognition (ASR) and Text-to-Speech (TTS) domains.
   * When using a Nvidia PyTorch container as the base, this is the recommended installation method for all domains.
```

Comment:
Change: "When using a Nvidia PyTorch container as the base, this is the recommended installation method for all domains." to:

When using an NVIDIA PyTorch container as the base, this is the recommended installation method for all domains.